src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/PropertiesShouldNotReturnArraysTests.cs

```diff
@@ -19,5 +19,56 @@ protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
             return new CSharpPropertiesShouldNotReturnArraysAnalyzer();
         }
+
+        [Fact]
+        public void TestCSharpPropertiesShouldNotReturnArraysArray()
```

Comment:
I am thinking of naming it 'TestCSharpPropertiesShouldNotReturnArraysWarning1', make more sense or not?